packages/nimble-components/src/table/tests/table-sorting.spec.ts

```diff
@@ -0,0 +1,689 @@
+import { html } from '@microsoft/fast-element';
```

Comment:
GitHub is subtly telling us that this is some great test coverage!
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/10500124/221632202-278c4a67-5f21-4535-8d3a-47b5a910d5b7.png">